Tribler/Test/Core/Modules/test_versioncheck.py

```diff
@@ -56,6 +57,20 @@ def assert_new_version_called(self, _):
     def check_version(self):
         return self.session.lm.version_check_manager.check_new_version().addCallback(self.assert_new_version_called)
 
+    def test_start(self):
+        """
+        Test whether the periodic version lookup works as expected
+        """
+        self.setup_version_server(json.dumps({'name': 'v1.0'}))
+        self.session.lm.version_check_manager.start()
+        self.assertFalse(self.session.lm.version_check_manager.is_pending_task_active("tribler version check"))
+
+        import Tribler.Core.Modules.versioncheck_manager as vcm
+        vcm.version_id = "7.0.0"
```

Comment:
Would be nice to have a test for a version with GIT in the version_id here.

Reply:
Actually, line 66 checks for this. Note that the `version_id` attribute is set to `7.0.0-GIT` in the `version.py` file.